src/modules/qt/gps_parser.h

```diff
@@ -29,20 +29,15 @@
 #include <time.h>
```

Comment:
It is a good practice to have as few includes in a header file as possible. I think that many of these are not used. Or are only being used by cpp files that include this header. I suggest to delete the includes that are not require by this header and move includes to cpp files if they are used there.

Reply:
Missed this in my final cleanup, done.